ambry-router/src/test/java/com.github.ambry.router/GetBlobInfoOperationTest.java

```diff
@@ -106,7 +104,7 @@ public GetBlobInfoOperationTest() throws Exception {
         CHECKOUT_TIMEOUT_MS, mockServerLayout, time);
     router = new NonBlockingRouter(new RouterConfig(vprops), new NonBlockingRouterMetrics(mockClusterMap),
         networkClientFactory, new LoggingNotificationSystem(), mockClusterMap, time);
-    blobProperties = new BlobProperties(BLOB_SIZE, "serviceId", "memberId", "contentType", false, Utils.Infinite_Time);
+    blobProperties = new BlobProperties(-1, "serviceId", "memberId", "contentType", false, Utils.Infinite_Time);
```

Comment:
Do you think it is a good idea? I feel that `BlobProperties` class has quite a few constructors due to the way it has evolved (changing that would touch many files). The static method feels a little out of place - BlobProperties class should not have to provide that method I feel. Perhaps `RouterUtils` could provide one. Will discuss this in person.